internals/models.py

```diff
@@ -359,6 +360,8 @@ def to_dict(self):
         output[key] = to_dict(value)
       elif isinstance(value, gae_users.User):
         output[key] = value.email()
+      elif isinstance(value, ndb.model.User):
+        output[key] = value.email()
       else:
```

Comment:
We need to add this case as unlike the not-so-old NDB Client Library, the Cloud NDB does not use GAE Users Library for creating `User` Objects

Reply:
OK.